File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.cs
public class Solution {
    public int MinimumLength(string s) {
        var n = s.Length;
        var freq = new int[26];
        foreach (var ch in s)
            freq[ch - 'a']++;
        
        var res = 0;
        for (var i = 0; i < n; i++) {
            if (freq[s[i] - 'a'] > 2)
                freq[s[i] - 'a'] -= 2;
        }

        foreach (var f in freq)
            res += f;
        return res;
    }
}

File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.c
int minimumLength(char* s) {
    int freq[26] = {};
    for (int i = 0; s[i]; i++)
        freq[s[i] - 'a']++;

    int res = 0;
    for (; *s; s++) {
        if (freq[*s - 'a'] > 2)
            freq[*s - 'a'] -= 2;
    }
    for (int i = 0; i < 26; i++)
        res += freq[i];
    return res;
}